src/components/EpisodePreview/EpisodePreview.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useContext } from "react";
+import {
+	EpisodesContext as Episodes
+} from "COMPONENTS/Episodes";
+import UTILS from "SHARED/utils.js";
+
+export default function EpisodePreview(props){
+
+	// CONTEXT
+	//-----------------------------
+	const {
+		get: {
+			episodeCount,
+			activeIndex,
+			overlayActive
+		},
+		set: {
+			activeIndex: setActiveIndex,
+			overlayActive: setOverlayActive
+		}
+	} = useContext(Episodes);
+
+	// RENDER
+	// ----------------------------
+	const { 
+		episodeId,
+		index,
+		number: episodeNo,
+		title,
+		thumbnail: {
+			src: thumbSrc,
+			description: thumbDesc
+		},
+		trailer: {
+			src: trailerSrc
+		}
+	} = props;
+
+	// handle visibility of active episode and episode preview overlay
+	const isActiveEpisode 	= activeIndex === index;
+	const thisOverlayActive = isActiveEpisode && overlayActive;
```

Comment:
oop - think I mentioend this before; this should be `const isOverlayActive`

Reply:
good spot, cheers